src/main/java/javapns/notification/Payload.java

```diff
@@ -74,7 +76,7 @@ public JSONObject getPayload() {
    * @throws JSONException
    */
   public void addCustomDictionary(final String name, final String value) throws JSONException {
-    logger.debug("Adding custom Dictionary [" + name + "] = [" + value + "]");
+    logger.debug(ADDING_CUSTOM_DICTIONARY + name + DELIMITER + value + "]");
```

Comment:
Can we make the DELIMITER DELIMITER_START and then add "]" as DELIMITER_END and not have one as a constant, but the other not?